profiles/desc/cpu_flags_arm.desc

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2017 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+aes - Use the AES cryptography instruction set
+crc32 - Use the CRC32 instruction set
+iwmmxt - Use the iwMMXt instruction set
+neon - Use the NEON (ASIMD) instruction set
+sha1 - Use the SHA-1 cryptography instruction set
+sha2 - Use the SHA-2 cryptography instruction set
+thumb2 - Enable Thumb-2 instruction set (ARMv6T2 and ARMv7+)
+v4 - Use ARMv4 instructions
+v4t - Use ARMv4T instructions
+v5t - Use ARMv5T instructions
```

Comment:
To name a precise version and variant of the ARM/Thumb architecture, the following strings are concatenated:
The string ARMv.
The version number of the ARM instruction set.
Variant letters of the included variants, except that the M variant is standard in architecture versions 4 and above, and therefore not normally listed.
If any variants described as standard in 3 above are not present, the letter x followed by the letters of the excluded variants. In addition, the letter P can be used after x to denote the exclusion of certain instructions from architecture version ARMv5TE, as described in The ARMv5TExP architecture version.